fix: Update cert instructions for NGINX One Console (#82)
nginx/documentation: content/includes/nginx-one/add-file/new-ssl-bundle.md

---
docs:
---

> First you can select the toggle to allow NGINX One Console to manaage the new certificate or bundle.

Reviewer: ```suggestion
First you can select the toggle to allow NGINX One Console to manage a new certificate or CA bundle.
```

In the screen that appears, you can add a certificate name. If you don't add a name, NGINX One will add a name for you, based on the expiration date for the certificate.

You can add certificates in the following formats:

- **SSL Certificate and Key**
- **CA Certificate Bundle**

In each case, you can upload files directly, or enter the content of the certificates in a text box. Once you upload these certificates, you may need to scroll down. You'll see:

- **Certificate Details**, with the Subject Name, start and end dates. 
- **Key Details**, with the encryption key size and algorithm, such as RSA

Select **Save and Continue**. You're taken to another screen where you can specify the locations for your files, which may be:

- **Certificate File Path**

>   - Enter the full path to your certificate, such as

Author: Change made in https://github.com/nginx/documentation/pull/82/files#r1943611417

>     - /etc/ssl/nginx/server.crt

Reviewer: ```suggestion
    - /etc/nginx/server.crt
```

>     - /etc/ssl/nginx/server.pem

Reviewer: ```suggestion
    - /etc/nginx/server.pem
```

>     - /etc/ssl/nginx/server.crt
>     - /etc/ssl/nginx/server.pem

Author: ```suggestion
    - /etc/nginx/server.crt
    - /etc/nginx/server.pem
```

- **Key File Path**
  - Enter the full path to your certificate key, such as

>     - /etc/nginx/server.key

Author: Updating
<img width="1368" alt="Screenshot 2025-02-05 at 1 12 40 PM" src="https://github.com/user-attachments/assets/ba94e532-96bb-4053-bfbc-39ca61dc7e90" />

>     - /etc/ssl/nginx/server.key

Reviewer: ```suggestion
    - /etc/nginx/server.key
```

>   - If you're using a `.pem` file, you won't have a separate key.

Reviewer: Feel free to correct me if I'm wrong. I don't think this is necessarily true. If it should be a certificate and key pair, we always require customers to upload a certificate file and a key file separately. For a CA certificate bundle, it would be a certificate file that only contains PEM blocks for public certificates.

Author: At least when I work through the UI, I don't see an option to include a separate key file. In that context, I'm thinking: 

```suggestion
  - If you're using a `.pem` file with private key information, you won't have a separate key file.
```

Reviewer 2: yep - if the user selected "SSL Certificate and Key" then the cert and key files need to be separate, and if they select "CA Certificate Bundle" then there is no private key provided, the CA bundle is just the certificates. The file format doesn't matter.

>   - If you're using a `.pem` file with private key information, you won't have a separate key file.

Author: <img width="1334" alt="Screenshot 2025-02-05 at 1 22 23 PM" src="https://github.com/user-attachments/assets/ad6fc6f5-227b-4113-8f83-86e453927a42" />

With the **Add Item** button, you can add the file to additional directories.

> When complete, select **Add** to include the certificate files that you've configured to desired directories.

Reviewer: ```suggestion
When complete, select **Add** to include the certificate and key files that you've configured to desired directories.
```
